src/common/components/hosts/HostStatus.tsx

```diff
@@ -92,8 +93,15 @@ const HostStatusPopoverContent: React.FC<HostStatusPopoverContentProps> = ({
           <>
             <br />
             {t(
-              "ai:To finish adding it to the cluster, approve its join request inside OpenShift Console's Nodes section. Note that it may take a few minutes for the join request to appear.",
+              "ai:To finish adding nodes to the cluster, approve the join request inside OpenShift Console's Nodes section.",
             )}
+            <br />
+            {t('ai:It may take a few minutes for the join request to appear.')}
+            <br />
+            {t('ai:If you prefer using the CLI, follow the instructions in')}&nbsp;
+            <ExternalLink href={APPROVE_NODES_IN_CL_LINK}>
+              {t('ai:How to approve nodes using the CL')}
```

Comment:
```suggestion
              {t('ai:How to approve nodes using the CLI')}
```
I think this may be a typo?

Reply:
Thanks. Definitely a type :slightly_smiling_face: